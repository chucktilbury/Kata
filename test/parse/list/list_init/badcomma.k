;-------------------------------------------------------------------------------
; verify wrong comma causes error
;-------------------------------------------------------------------------------

var list name = [123 x 123, 123]
