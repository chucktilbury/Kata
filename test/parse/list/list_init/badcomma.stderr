Syntax: /home/chuck/Src/Kata/test/parse/list/list_init/badcomma.k: 5: 23: expected a ',' or a ']' but got symbol
Syntax: /home/chuck/Src/Kata/test/parse/list/list_init/badcomma.k: 5: 27: expected an assignment item but got [
Syntax: /home/chuck/Src/Kata/test/parse/list/list_init/badcomma.k: 5: 27: expected module item or end of file but got var
Syntax: /home/chuck/Src/Kata/test/parse/list/list_init/badcomma.k: 5: 27: module cannot be empty
